webui/src/app/components/appdetail/appdetail.component.ts

```diff
@@ -7,10 +7,11 @@ import { Store } from '@ngrx/store';
 import { ApplicationsStoreService } from '../../stores/applications-store.service';
 import { EnvironmentsStoreService } from '../../stores/environments-store.service';
 import { element } from 'protractor';
-import { ISubscription } from 'rxjs/Subscription';
+import { SubscriptionLike as ISubscription } from 'rxjs';
 import { AutoUnsubscribe } from '../../shared/decorator/autoUnsubscribe';
 import { BadgeShieldsIOBean } from '../../widget/badgewidget/badgewidget.component';
 import {FieldsetModule} from 'primeng/fieldset';
+import { Observable } from 'rxjs/internal/Observable';
```

Comment:
Sounds very weird to rely on someone else' internals! Auto-import typo for `rxjs/Observable`?